src/common.js

```diff
@@ -447,7 +447,11 @@ function wrap(cmd, fn, options) {
         // Perform glob-expansion on all arguments after globStart, but preserve
         // the arguments before it (like regexes for sed and grep)
         if (!config.noglob && options.allowGlobbing === true) {
-          args = args.slice(0, options.globStart).concat(expand(args.slice(options.globStart)));
+          var globStart =
+            options.globStart instanceof Function
```

Comment:
Is this strictly necessary? We were able to implement `head -n <num>` without this. See https://github.com/shelljs/shelljs#head-n-num-file--file-

Reply:
I'll check it out!

Reply:
The latest changes have been pushed which revert `globStart` back to normal. Thanks for linking to `head` so I could figure it out from that!